stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  template:
+    metadata:
+      name: {{ template "fullname" . }}
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
Having these labels in the Pod template can cause issues when upgrade the DaemonSet as the selector changes. You should either only keep `app` or set a custom selector.